app/models/form/sales/questions/prevshared.rb

```diff
@@ -0,0 +1,17 @@
+class Form::Sales::Questions::Prevshared < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "prevshared"
+    @check_answer_label = "Buyers previously owned a shared property"
```

Comment:
should this be Buyer/Buyers depending if it's a joint purchase? or does this only occcur for shared ownerships already